### ISSUE TITLE
A Suggestion 

### ISSUE THREAD

**Issue by ceballos59** [pencilblue/pencilblue#1192]
I have noticed that if you change in all <div>.. data-as-sortable = "**dragControlListeners**" ... </div> instead of "**board.dragControlListener**

then

$scope.dragControlListeners = {
    accept: function (sourceItemHandleScope, destSortableScope) {return boolean}//override to determine drag is allowed or not. default is true.
    itemMoved: function (event) { // Do what you want},
    orderChanged: function(event) { // Do what you want},
    containment: '#board'//optional param.
    clone: true //optional param for clone feature.
    allowDuplicates: false //optional param allows duplicates to be dropped.
};

Works perfectly. 

With **board.dragControlListener** , does not work.

**Comment by blakecallens** [comment]
Thanks for the catch, @ceballos59. Would you be willing to put a pull request in?

**Comment by ceballos59** [comment]
I'm sorry but my working version in pencilblue is completely disconnected from the standard, so to add a pull request, you would have to enable a fork previously. Is it so? Could you explain the steps I must take to comply with your request in this case.

**Comment by dukeofsussex** [comment]
@ceballos59 Yes, you would need to fork this repo, clone the forked one, apply your changes, push to your fork and the submit a PR with those changes
